From 4dde3669ad5bf9ab52a01993d392481690513409 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:08 -0400
Subject: [redhat] kabi: Adding symbol seq_lseek (fs/seq_file.c)

Message-id: <0e14fb86100799e6bf90202dab8a70ca447509e0.1410510773.git.poros@redhat.com>
Patchwork-id: 96007
O-Subject: [PATCH RHEL7.1 188/233] kabi: Adding symbol seq_lseek (fs/seq_file.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Alexander Viro <aviro@redhat.com>

Bugzilla: 1071099,1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    The function is used as the .llseek method for the seek operation on the proc interface.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/seq_lseek
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/seq_lseek
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/seq_lseek

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/seq_lseek  b/redhat/kabi/kabi-module/kabi_ppc64/seq_lseek 
new file mode 100644
index 0000000..2b70e32
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/seq_lseek  
@@ -0,0 +1,2 @@
+#1-
+0x8a961231 seq_lseek vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/seq_lseek  b/redhat/kabi/kabi-module/kabi_s390x/seq_lseek 
new file mode 100644
index 0000000..5ecac4d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/seq_lseek  
@@ -0,0 +1,2 @@
+#1-
+0x57114dba seq_lseek vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/seq_lseek  b/redhat/kabi/kabi-module/kabi_x86_64/seq_lseek 
new file mode 100644
index 0000000..db7982f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/seq_lseek  
@@ -0,0 +1,2 @@
+#1-
+0x1685c91c seq_lseek vmlinux EXPORT_SYMBOL
-- 
1.7.1